index.bs

```diff
@@ -2693,10 +2693,16 @@ WritableStream(<var>underlyingSink</var> = {}, { <var>size</var>, <var>highWater
   behavior will be the same as a {{CountQueuingStrategy}} with a <a>high water mark</a> of 1.
 </div>
 
+<div class="note">
+  The \[[inFlightCloseRequest]] slot and \[[closeRequest]] slot are mutually exclusive. No element will be removed from
+  \[[writeRequests]] while \[[inFlightWriteRequest]] is not <code>undefined</code>. Implementations may optimize the way
```

Comment:
English nit:

"Implementations may optimize storage for these slots" would sound more natural.